autoscheduler/scraper/tests/api_tests.py

```diff
@@ -196,7 +249,7 @@ def test_api_sections_gives_valid_response(self):
                         'type': 'LEC',
                     },
                     {
-                        'id': '20',
+                        'id': '11',
```

Comment:
What's the reason for this ID change?

Reply:
The previous ID was incorrect, so the test was failing even when it should pass